tools/ark-cli/src/lib/executeQuery.ts

```diff
@@ -207,6 +215,85 @@ async function executeQueryWithFormat(options: QueryOptions): Promise<void> {
   }
 }
 
+async function watchEventsLive(queryName: string): Promise<void> {
```

Comment:
would this potentially be better in the kube client code we have, i think there's one file where we're moving the `kubectl` exec commands to [which I'm sure at a later date would make it easier to convert them to simply use the k8s client sdk]